### ISSUE TITLE
psql-init.py fails with ckan 2.9.1

### ISSUE THREAD

**Issue by matthiasdg** [keitaroinc/ckan-helm#13]
Was looking for a ckan helm chart and seems yours is the most up to date, tnx!
I noticed the psql-init job does not complete successfully; it crashes when setting the datastore permissions (`/usr/bin/ckan -c /srv/app/production.ini datastore set-permissions`). 

After looking into it, I noticed errors like:
```
ERROR [ckan.lib.search.common] Failed to connect to server at 'http://127.0.0.1:8983/solr/select/?q=%2A%3A%2A&rows=1&wt=json'
,are you sure that URL is correct?
Checking it in a browser might help: HTTPConnectionPool(host='127.0.0.1', port=8983):
Max retries exceeded with url: /solr/select/?q=%2A%3A%2A&rows=1&wt=json
(Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fad2b589a30>:
Failed to establish a new connection: [Errno 111] Connection refused'))
```
then a bit further:
```
During handling of the above exception, another exception occurred:
...
  File "/srv/app/src/ckan/ckanext/datastore/backend/__init__.py", line 90, in set_active_backend
    schema = config.get(u'ckan.datastore.write_url').split(u':')[0]
AttributeError: 'NoneType' object has no attribute 'split'
```

I could prevent the crash by adding `CKAN_SOLR_URL` and `CKAN_DATASTORE_WRITE_URL` to `templates\psql-init-job.yaml`. (The solr error does not seem to cause the crash; it works without the solr url too, the datastore one is though).
It is a bit confusing to then have `CKAN__DATASTORE__WRITE_URL` (which is used elsewhere in the psql init apparently) and `CKAN_DATASTORE_WRITE_URL` (fewer underscores, used by the ckan binary) but worked.

After this, I still got an error for `re.sub("\\\connect.*", "", sql)`, since sql contains bytes, so I first did a `.decode('utf-8')` on that.
Then it was successful.

**Comment by mbocevski** [comment]
@matthiasdg Resolved in #15. The issue with the naming of the env variables is due to how they are handled in CKAN 2.7/2.8 vs 2.9. With #15 the chart will use 2.9 style environment variables. Please give it a try and let us know if everything is working fine.

**Comment by matthiasdg** [comment]
Whoa, that was fast! Tnx! Just tried it and seems to work fine!
